packages/odyssey-storybook/src/components/odyssey-labs/SideNav/SideNav.stories.tsx

```diff
@@ -347,19 +330,20 @@ export const Default: StoryObj<SideNavProps> = {
       }
     });
     await step("Side Nav Collapse", async ({}) => {
-      const collapseButton = within(expandedRegion).getByLabelText(
+      const collapseButton = within(collapsedRegion).getByLabelText(
         "collapse side navigation",
       );
       await userEvent.click(collapseButton);
       await waitFor(() => {
         expect(expandedRegion).not.toBeVisible();
-        expect(collapsedRegion).toBeVisible();
       });
     });
     await step("Side Nav Expand", async ({}) => {
-      await userEvent.click(collapsedRegion);
+      const collapseButton = within(collapsedRegion).getByLabelText(
+        "expand side navigation",
+      );
+      await userEvent.click(collapseButton);
```

Comment:
Is there a role, implicit or explicit, associated with this collapse button?

```suggestion
      const collapseButton = within(collapsedRegion).getByRole("button", {
        name: "expand side navigation",
      });
      await userEvent.click(collapseButton);
```